src/atomate2/vasp/flows/defect.py

```diff
@@ -0,0 +1,224 @@
+"""Flows used in the calculation of defect properties."""
+
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass, field
+from typing import Iterable
+
+from jobflow import Flow, Maker, OutputReference, job
+from pymatgen.core.structure import Structure
+
+from atomate2.vasp.jobs.base import BaseVaspMaker
+from atomate2.vasp.jobs.core import RelaxMaker, StaticMaker
+from atomate2.vasp.jobs.defect import (
+    FiniteDifferenceMaker,
+    get_ccd_from_task_docs,
+    spawn_energy_curve_calcs,
+)
+from atomate2.vasp.schemas.defect import CCDDocument
+from atomate2.vasp.sets.core import StaticSetGenerator
+from atomate2.vasp.sets.defect import AtomicRelaxSetGenerator
+
+logger = logging.getLogger(__name__)
+
+
+DEFAULT_DISTORTIONS = (-1, -0.15, -0.1, -0.05, 0, 0.05, 0.1, 0.15, 1)
+
+DEFECT_RELAX_GENERATOR = AtomicRelaxSetGenerator(use_structure_charge=True)
```

Comment:
You can remove the use_structure_charge here because its specified in the input set itself.

Reply:
I just wantted to be explicit here incase anything upstream changes since this is the most important setting